src/figuresClasses.ts

```diff
@@ -1,19 +1,99 @@
-export interface Figure {
 
+export type Shape = 'triangle' | 'circle' | 'rectangle';
+export type Color = 'red' | 'green' | 'blue';
+
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea: () => number;
 }
 
-export class Triangle {
+export const roundedDownToHundredths = (num: number): number => {
+  return +(Math.floor(num * 100) / 100);
+};
```

Comment:
Why did you use + here?

Reply:
I forgot how work roundet to ... , i think if Math.floor(3.10011* 100) / 100 = 3.100 , so i add + for delete 0 at the end